### ISSUE TITLE
BillboardChart Thresholds set to 0 by default

### ISSUE THREAD

**Issue by edevregille** [newrelic/terraform-provider-newrelic#1432]
**Terraform Version**

```
terraform {
  required_version = ">= 0.15.0"
  required_providers {
    newrelic = {
      source  = "newrelic/newrelic"
      version = "~> 2.25.0"
    }
  }
}

```

**Affected Resource(s)**

Dashboard -> BillboardChart

**Terraform Configuration**

```
dynamic "widget_billboard" {
                for_each = page.value["BillboardChart"]
                content {
                    title = widget_billboard.value["title"]
                    row = widget_billboard.value["row"]
                    column = widget_billboard.value["column"]
                    width = widget_billboard.value["width"]
                    height = widget_billboard.value["height"]
                    nrql_query{
                        account_id = var.accountId
                        query = widget_billboard.value["nrql"]
                    }
                }
            }

```

**Actual Behavior**

The provider sets the critical and warning optional fields to 0 when no value is provided. 
Result is that the chart shows the Billboard values as critical (red) as soon as value is >0.

**Expected Behavior**

If there is no value provided it should be not setting any threshold. Or it should provide a way to set it to no threshold (tried setting critical and warning to null in the terraform but got same result).

**Comment by bgillett** [comment]
Im also running into this same bug; so please don't close this issue.

**Comment by agcorrea** [comment]
Waiting for this fix as well.

**Comment by EdwardAndrew** [comment]
Also waiting for this fix.

**Comment by SteeltownRiot** [comment]
Having the same problem.

+1 to @plimpton 's suggestion NULL should be the default, not 0

**Comment by JoannaFyson** [comment]
I agree, default should be NULL, to prevent any colour-coding.  Has anyone discovered how to configure in terraform to null?  I've tried:
warning = "None"
warning = None
warning =
all of which throw errors when applying the plan

**Comment by brian-labelle** [comment]
I am running into the same issue and have not yet been able to identify a workaround

**Comment by kidk** [comment]
There's a pull request open to resolve this issue #1528 Thanks everyone for your patience

**Comment by kidk** [comment]
Fix has gone out with v2.34.1. Closing this for new, don't hesitate to open a new ticket if you encounter any problems.
